data/chriskalmar.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Chris Kalmar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Senior Software Engineer from Vienna",
+  "avatar": "https://github.com/chriskalmar.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/chriskalmar",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/chriskalmar",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on YouTube",
+      "url": "https://www.youtube.com/channel/UCHQ_4Fl3ErPDfFutizT2B_g",
```

Looks good to merge, Thank you for your contribution! 🎉

nonBlocking: if you wanted to use your youtube handle in url, more clear one! 🙂

```suggestion
      "url": "https://www.youtube.com/@chriskalmar5046",
```

---

wow, I didn't know that this even works with Youtube.
I thought you need 1000 subscribers or so.
Thanks @kumarsonsoff3 🙏

---

No-no YouTube has recently changed this and allows everyone to secure their handles! 😉